tests/user_examples/test_2d_water_entry_exit/test_2d_water_entry_exit.cpp

```diff
@@ -0,0 +1,539 @@
+﻿/**
+ * @file	water entry and exit.cpp
+ * @brief	2D water entry and exit example with surface wettability considered.
+ * @details	This is the one of FSI test cases, also one case for
+ * 			understanding spatial temporal identification approach, 
+ *          especially when coupled with the wetting.
+ * @author  Shuoguo Zhang and Xiangyu Hu
+ */
+#include "sphinxsys.h" //SPHinXsys Library.
+#include "wetting_coupled_spatial_temporal_method.h"
+
+using namespace SPH;   // Namespace cite here.
+//----------------------------------------------------------------------
+//	Basic geometry parameters and numerical setup.
+//----------------------------------------------------------------------
+Real insert_cylinder_radius = 0.055;                              /**< Cylinder radius. */
+Real DL = 8.0 * insert_cylinder_radius;                          /**< Water tank length. */
+Real DH = 7.0 * insert_cylinder_radius;                          /**< Water tank height. */
+Real LL = DL;                                                     /**< Water column length. */
+Real LH = 3.0 * insert_cylinder_radius;                           /**< Water column height. */
+Real particle_spacing_ref = 2.0 * insert_cylinder_radius / 40.0;  /**< Initial reference particle spacing. */
+Real BW = particle_spacing_ref * 4;                               /**< Thickness of tank wall. */
+Vec2d insert_cylinder_center(0.5 * DL, LH + 0.15);                /**< Location of the cylinder center. */
+Vecd tethering_point(0.5 * DL, DH);                               /**< The tethering point. */
+StdVec<Vecd> observer_location = {insert_cylinder_center};        /**< Observation point. */
+//----------------------------------------------------------------------
+//	Material parameters.
+//----------------------------------------------------------------------
+Real rho0_f = 1.0;                       /**< Fluid density. */
+Real rho0_s = 0.5;                       /**< Cylinder density. */
+Real gravity_g = 9.81;                   /**< Gravity. */
+Real U_max = 2.0 * sqrt(gravity_g * LH); /**< Characteristic velocity. */
+Real c_f = 10.0 * U_max;                 /**< Reference sound speed. */
+Real mu_f = 8.9e-7;                      /**< Water dynamics viscosity. */
+//----------------------------------------------------------------------
+//	Wetting parameters
+//----------------------------------------------------------------------
+Real diffusion_coff = 330.578 * pow(particle_spacing_ref,2); /**< Wetting coeffecient. */
```

Comment:
```suggestion
Real diffusion_coeff = 330.578 * pow(particle_spacing_ref,2); /**< Wetting coefficient. */
```

Reply:
thanks for your check about this typo, I will modifiy it

Reply:
I have modified these typos, please check